packages/tasit-action/src/contract/Action.js

```diff
@@ -77,7 +77,10 @@ export class Action extends Subscription {
     const baseEthersListener = async blockNumber => {
       try {
         if (!this.#tx) this.#tx = await this.#txPromise;
-
+        
+        if (this.#tx.status == 0) {
+          throw new Error(`Transaction has failed and it's status is 0`);
```

Comment:
The error should be emitted to `errorListener` to be caught by the user instead of `throw` as an exception.
Use: `this._emitErrorEventFromEventListener()` for that.